server/catgenome/src/main/resources/database/catgenome/h2/v2022.02.22_16.00__pathway_database_source.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE CATGENOME.PATHWAY ADD COLUMN DATABASE_SOURCE BIGINT;
```

Comment:
Let's add default value matching `CUSTOM` source (to init all existing pathways)